### ISSUE TITLE
Maintenance of project?

### ISSUE THREAD

**Issue by ionelmc** [peritus/bumpversion#169]
Hey @peritus can you give an update about your plans about this project? 

It appears that @c4urself has an active fork with pypi releases (as `bump2version`). 

Perhaps a transition can be worked out?

**Comment by peritus** [comment]
@ionelmc

I should probably add a sunsetting notice to the top of the README, saying this is not actively maintained and people should switch to one of the forked versions.

The other project I'm aware of is @vortec's `versionbump` (https://pypi.python.org/pypi/versionbump / https://github.com/vortec/versionbump) that looks like a nice replacement.

**Comment by ionelmc** [comment]
@peritus yes but this project is more complete. Perhaps consider conceding maintenance to someone?

**Comment by ionelmc** [comment]
@peritus So I take that's a no. Not even giving pypi upload permissions?

**Comment by anthrotype** [comment]
@peritus Yes, please! The only reason a `bump2version` exists is to be able to upload it to PyPI under a different name.

**Comment by vEpiphyte** [comment]
Any updates on transferring the maintenance of this project to someone else?

**Comment by sharksforarms** [comment]
any updates? ..

**Comment by deeplook** [comment]
Any updates?

**Comment by sharksforarms** [comment]
@deeplook https://github.com/c4urself/bump2version

**Comment by sumpfralle** [comment]
What a pity, that also the [Debian package](https://packages.debian.org/bumpversion) of this helpful piece of software stalls due to the uncertainty :(

I just filed a [wishlist bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=916498) asking the maintainer of the Debian package for considering [c4urself`s fork](https://github.com/c4urself/bump2version) for future package updates.

@peritus: it would be so nice and reduce the pain and additional work so much, if you could manage to let this great project continue to blossom in the hands of fellow (co-)maintainers.

**Comment by tjanez** [comment]
This could get resolved pretty soon. See: https://github.com/c4urself/bump2version/issues/86#issuecomment-530106531.
